shims/common/src/main/scala/org/apache/gluten/GlutenConfig.scala

```diff
@@ -2043,4 +2045,12 @@ object GlutenConfig {
       .doubleConf
       .checkValue(v => v >= 0 && v <= 1, "offheap sizing memory fraction must between [0, 1]")
       .createWithDefault(0.6)
+
+  val CELEBORN_FALLBACK_ENABLED =
+    buildStaticConf("spark.gluten.sql.columnar.shuffle.celeborn.fallback.enabled")
+      .internal()
+      .doc("Enable celeborn fallback. Throw an exception immediately" +
```

Comment:
Suggestion:

If enabled, fall back to ColumnarShuffleManager when celeborn service is unavailable. Otherwise, throw an exception.